docs/src/index.md.template

```diff
@@ -32,7 +32,7 @@ the popular [`Tulip.jl`](https://github.com/ds4dm/Tulip.jl),
 
 ```@raw html
 <div align="center">
-<img style="width:300px;margin:10px;border-offset:15px;border: 1px solid #eee;border-radius: 50%;padding: 10px;-webkit-border-radius: 50%;-moz-border-radius: 50%;" src="https://git-r3lab.uni.lu/lcsb-biocore/COBREXA.jl/-/jobs/artifacts/master/raw/output.gif?job=generator:gource" alt="history"><br>
+<img style="width:300px;margin:10px;border-offset:15px;border: 1px solid #eee;border-radius: 20%;padding: 10px;" src="https://git-r3lab.uni.lu/lcsb-biocore/COBREXA.jl/-/jobs/artifacts/master/raw/output.gif?job=gource" alt="history"><br>
```

Comment:
how does it look like? Can you also change the gitlab url?

Reply:
It's IMO better (doesn't kill 30% of the nice picture area, just like 5%):
![image](https://user-images.githubusercontent.com/271543/139815126-e8238fc5-21d6-4ce8-a64a-86f6336cb9ef.png)

The URL got changed because the step name changed. (I hope the prefix wasn't there just because of this.)

Reply:
hmmm, the idea was to keep a circle, like a lens

Reply:
ah so. I kinda imagined a submarine window so far.... :D

Reply:
not sure why we have to change :D